core/platform/taskqueue/dev_mode_taskqueue_services_test.py

```diff
@@ -77,7 +91,17 @@ def test_task_handler_will_create_the_correct_post_request(self):
             'X-AppEngine-Fake-Is-Admin': '1',
             'method': 'POST'
         }
-        def mock_post(url, json, headers, timeout):
+        # In the type annotation below, we have used Dict[str, Any] for json.
+        # This is because this function mocks requests.post function where the
+        # type of json has been defined Any, hence using Dict[str, Any] here.
+        # https://github.com/python/typeshed/blob/master/stubs/requests/requests/api.pyi#L78.
+
```

Comment:
```suggestion
        # In the type annotation below, we have used Dict[str, Any] for JSON.
        # This is because this function mocks requests.post function where the
        # type of JSON has been defined Any, hence using Dict[str, Any] here.
        # https://github.com/python/typeshed/blob/master/stubs/requests/requests/api.pyi#L78.
```

Reply:
(remove the line between the method call and the comment)

Reply:
Done